MarkBind/markbind
# Remove extra line

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x ] Documentation update

Fixes #837 

**What is the rationale for this request?**
There was an extra line in the user guide

**What changes did you make? (Give an overview)**
Removed the extra line

**Testing instructions:**
1. Open user guide
2. Navigate to Authoring Contents > Formatting Contents > Horizontal Rule
3. Check that there are 3 lines, as expected
